app/views/admin/edition_translations/edit.html.erb

```diff
@@ -17,19 +17,25 @@
     </section>
   </div>
   <div class="col-md-4">
-    <%= sidebar_tabs edition_tabs(
-      @edition,
-      remarks_count: @edition_remarks.length,
-      history_count: @edition_history.total_count,
-      editing: true
-    ) do |tabs| %>
-
-      <%= tabs.pane class: "govspeak_help", id: "govspeak_help" do %>
+    <% if current_user.can_view_move_tabs_to_endpoints? %>
+      <div class="govspeak_help" id="govspeak_help">
         <%= render partial: "admin/editions/govspeak_help",
                    locals: { hide_inline_attachments_help: !@edition.allows_inline_attachments?, show_attachments_tab_help: true } %>
-      <% end %>
+      </div>
+    <% else %>
+      <%= sidebar_tabs edition_tabs(
+        @edition,
+        remarks_count: @edition_remarks.length,
+        history_count: @edition_history.total_count,
+        editing: true,
+        user_has_view_move_tabs_to_endpoints_permission: current_user.can_view_move_tabs_to_endpoints?
```

Comment:
Is the `user_has_view_move_tabs_to_endpoints_permission` param still needed?

Reply:
Ah sorry they snuck back in when i rebased. Thanks!